src/storage/CommonUtils.h

```diff
@@ -0,0 +1,130 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef STORAGE_COMMON_H_
+#define STORAGE_COMMON_H_
+
+#include "base/Base.h"
+#include "filter/Expressions.h"
+
+namespace nebula {
+namespace storage {
+
+class CommonUtils {
+public:
+    static std::string buildTagKeyInFilter(const std::string& tag, const std::string& prop) {
+        return folly::stringPrintf("%s_%s", tag.c_str(), prop.c_str());
```

Comment:
I test that we can use “_” in create tag, so “a”+”b_c” is same with “a_b”+”c”

Reply:
Good catch.